sous-chef/tests/integration_tests/data/final_grocery_list.csv

```diff
@@ -1,49 +1,48 @@
+tbsp,tablespoon,paprika powder,False,0.5,True,Spices and herbs,paprika powder,Amazon,,[Spanish Deluxe Potatoes],2022-01-23 18:15:00+00:00,[Sun],False,Amazon
+ml,milliliter,passata,False,350.0,True,Sauces,passata,grocery store,4311501650578,[Lasagna],2022-01-25 17:15:00+00:00,[Tue],False,Sauces cans and oils
+tsp,teaspoon,pepper,False,1.35,True,Spices and herbs,pepper,Asian store,,"[Lasagna, Spanish Deluxe Potatoes]",2022-01-23 18:15:00+00:00,"[Sun, Tue]",False,Asian store
+g,gram,potato,False,500.0,True,Vegetables,potatoes,grocery store,,[Spanish Deluxe Potatoes],2022-01-23 18:15:00+00:00,[Sun],False,Farmland pride
+,,potato,False,2.0,True,Vegetables,potatoes,grocery store,,[Lasagna],2022-01-25 17:15:00+00:00,[Tue],False,Farmland pride
+g,gram,pumpkin puree,False,100.0,True,Canned,pumpkin puree,grocery store,4260181088037,[Holiday cheesecake],2022-01-23 11:30:00+00:00,[Sun],False,Sauces cans and oils
+tbsp,tablespoon,rapseed oil,False,2.5,True,Fats and oils,rapseed oil,grocery store,,"[Baked Oatmeal Bars, Spanish Deluxe Potatoes]",2022-01-23 18:15:00+00:00,[Sun],False,Sauces cans and oils
+tbsp,tablespoon,red wine vinegar,False,0.5,True,Sauces,red wine vinegar,grocery store,4008100120068,[Lasagna],2022-01-25 17:15:00+00:00,[Tue],False,Sauces cans and oils
+cup,cup,rice flakes,False,0.75,True,Grains,rice flakes,grocery store,,[Baked Oatmeal Bars],2022-01-23 18:15:00+00:00,[Sun],False,Bulk
+g,gram,ricotta,False,125.0,True,Dairy products,ricotta,grocery store,,[Lasagna],2022-01-25 17:15:00+00:00,[Tue],False,Dairy force
+,,ripe banana,False,3.0,True,Fruits,ripe bananas,grocery store,,[Baked Oatmeal Bars],2022-01-23 18:15:00+00:00,[Sun],False,Farmland pride
+tsp,teaspoon,salt,False,0.75,True,Spices and herbs,salt,grocery store,,"[Baked Oatmeal Bars, Spanish Deluxe Potatoes]",2022-01-23 18:15:00+00:00,[Sun],False,Spices and herbs
+cup,cup,shredded gratin cheese,False,0.25,True,Dairy products,shredded gratin cheese,grocery store,5760466896467,[Lasagna],2022-01-25 17:15:00+00:00,[Tue],False,Dairy force
+cup,cup,sour cream,False,0.3,True,Dairy products,sour cream,grocery store,4046700014301,[Holiday cheesecake],2022-01-23 11:30:00+00:00,[Sun],False,Dairy force
+cup,cup,sugar,False,0.5,True,Baking,sugar,grocery store,4006544205006,[Holiday cheesecake],2022-01-23 11:30:00+00:00,[Sun],False,Sweet carolina
+tsp,teaspoon,vanilla extract,False,2.0,True,Baking,vanilla extract,Homemade,,"[Baked Oatmeal Bars, Holiday cheesecake]",2022-01-23 11:30:00+00:00,[Sun],False,Homemade
+cup,cup,whey protein powder,False,0.25,True,Spices and herbs,whey protein powder,Amazon,683442920,[Baked Oatmeal Bars],2022-01-23 18:15:00+00:00,[Sun],False,Amazon
+can,can,whole tomatoes,False,1.0,True,Canned,whole tomatoes,grocery store,4311596062621,[Lasagna],2022-01-25 17:15:00+00:00,[Tue],False,Sauces cans and oils
+tbsp,tablespoon,yogurt,False,2.0,True,Dairy products,yogurt,grocery store,4062800016151,[Spanish Deluxe Potatoes],2022-01-23 18:15:00+00:00,[Sun],False,Dairy force
```

Comment:
@AkagiShigeru any ideas why times changed? I.e. last entry went from 17:45 to 18:15. Wasn't the case in other or, was just a store change

Reply:
@AkagiShigeru maybe re-check out main. I didn't change anything except rebase off main, and this resolved itself.
see https://github.com/ArBridgeman/aurorianclouds/blob/CODE-209/sous-chef/tests/integration_tests/data/final_grocery_list.csv